lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "8.76.6",
+  "version": "9.0.0-next.22",
```

Comment:
We'll update this file to remove references to `next` once we're ready to publish v9, correct?